test/e2e/lib/gutenberg/tracking/general-tests.js

```diff
@@ -158,6 +158,32 @@ export function createGeneralTests( { it, editorType, postType } ) {
 		);
 	} );
 
+	it( 'Tracks "wpcom_block_editor_nav_sidebar_open" event', async function () {
+		const editor = await EditorComponent.Expect( this.driver, gutenbergEditorType );
+
+		// We open and close the navigation sidebar to make sure we are only tracking the open.
+		await editor.toggleNavigationSidebar();
+		await editor.toggleNavigationSidebar();
+
+		const eventsStack = await getEventsStack( this.driver );
+		const navSidebarOpenFiredOnce =
+			eventsStack.filter( ( [ eventName ] ) => eventName === 'wpcom_block_editor_nav_sidebar_open' )
+				.length === 1;
+		const editorCloseClickNotFired =
+			eventsStack.filter( ( [ eventName ] ) => eventName === 'wpcom_block_editor_close_click' )
+				.length === 0;
+		assert.strictEqual(
```

Comment:
Should we add a second test to ensure this fires as expected on `< All Pages` / `< All Posts` / `< Dashboard` ?

Reply:
Any idea how to tackle this? I've the tests written (locally) but clicking on the button closes the editor. Which removes the E2E events stack.

Reply:
Ohh good point.  Im not quite sure the best way to handle that.  Maybe a simple way would be to define the selector for this tracks event as a constant, use that import in both the test and the tracks event, and have the test just verify that selector is present where we would expect it to be?  It wouldn't technically test that the event is firing, but it would at least notify us if the selector that event is tied to is no longer present 🤔 .

Reply:
@WunderBart Any idea? Is there a way to disable/prevent navigations? Googled it but no luck 😞

Reply:
Not sure what's the issues here, could you elaborate please?

Reply:
Actually, should we even test it here? Isn't that something we should do in ETK?